iphone/Maps/LocalizedStrings/en-GB.lproj/LocalizableTypes.strings

```diff
@@ -528,6 +528,7 @@
 "type.highway.services" = "Service Area";
 "type.highway.speed_camera" = "Speed Camera";
 "type.highway.steps" = "Stairs";
+"type.highway.ladder" = "Ladder";
```

Comment:
i'm not 100% sure, but i think only the `en` string should be added (weblate will handle the others?)